src/control/cmd/daos/util.h

```diff
@@ -24,3 +27,86 @@
 #include "daos_hdlr.h"
 
 int resolve_duns_path(struct cmd_args_s *ap);
+
+/* cgo is unable to work directly with preprocessor macros
+ * so we have to provide these glue helpers.
+ */
+static inline uint64_t
+daos_prop_co_status_val(uint32_t status, uint32_t flag, uint32_t ver)
+{
+	return DAOS_PROP_CO_STATUS_VAL(status, flag, ver);
+}
+
+/* cgo is unable to work directly with unions, so we have
+ *  to provide these glue helpers.
+ */
+static inline char *
+get_dpe_str(struct daos_prop_entry *dpe)
+{
+	if (dpe == NULL)
+		return NULL;
+
+	return dpe->dpe_str;
+}
+
+static inline uint64_t
+get_dpe_val(struct daos_prop_entry *dpe)
+{
+	if (dpe == NULL)
+		return 0;
+
+	return dpe->dpe_val;
+}
+
+static inline void *
+get_dpe_val_ptr(struct daos_prop_entry *dpe)
+{
+	if (dpe == NULL)
+		return NULL;
+
+	return dpe->dpe_val_ptr;
+}
+
+static inline void
+set_dpe_str(struct daos_prop_entry *dpe, d_string_t str)
+{
+	if (dpe == NULL)
+		return;
+
+	dpe->dpe_str = str;
+}
+
+static inline void
+set_dpe_dupe_str(struct daos_prop_entry *dpe, d_string_t str, int strlen)
```

Comment:
probably OK but there's a namespace conflict on "strlen" here, there's a chance it'll be interpreted as a function pointer rather than a int.  It should probably also be a size_t, or daos_size_t strictly.

Reply:
Fixed.